.github/workflows/main.yml

```
@@ -16,7 +16,15 @@ jobs:
       with:
         node-version: ${{ matrix.node-version}}
         cache: 'npm'
+
     - name: Installing dependencies
-      run: |
-        npm install
-        npm ci
+      run: npm install
```

```suggestion
      run: npm ci
```

This should be `npm ci` : https://support.deploybot.com/article/131-why-developers-should-use-npm-ci-instead-of-npm-install-and-its-benefits